lib/erb_lint/linters/rubocop.rb

```diff
@@ -74,6 +72,37 @@ def format_error(token, offense)
           message: offense.message.strip
         }
       end
+
+      def config_from_hash(hash)
+        inherit_from = hash.delete('inherit_from')
+        resolve_inheritance(hash, inherit_from)
+
+        tempfile_from('.erblint-rubocop', hash.to_yaml) do |tempfile|
+          RuboCop::ConfigLoader.load_file(tempfile.path)
+        end
+      end
+
+      def resolve_inheritance(hash, inherit_from)
+        base_configs(inherit_from)
+          .reverse_each do |base_config|
+          base_config.each do |k, v|
+            hash[k] = hash.key?(k) ? RuboCop::ConfigLoader.merge(v, hash[k]) : v if v.is_a?(Hash)
+          end
+        end
+      end
+
+      def base_configs(inherit_from)
+        regex = URI::DEFAULT_PARSER.make_regexp(%w[http https])
+        configs = Array(inherit_from).compact.map do |base_name|
+          if base_name =~ /\A#{regex}\z/
+            RuboCop::ConfigLoader.load_file(RuboCop::RemoteConfig.new(base_name, Dir.pwd))
+          else
+            config_from_hash(@file_loader.yaml(base_name))
```

Comment:
This is the main reason from not using rubocop's inheritance mechanism, going through file_loader means we can load files from github in policial. This is based on https://github.com/bbatsov/rubocop/blob/60211ad3e0d5911d56d5909b248ae28a92138f06/lib/rubocop/config_loader_resolver.rb#L85-L103